packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -185,6 +186,11 @@ export default class BackupNg {
           vmIds.forEach(handleRecord)
           unboxIdsFromPattern(job.srs).forEach(handleRecord)
 
+          // add xapi specific to the healthcheck SR if needed
+          if (job.settings[schedule.id].healthCheckSr !== undefined) {
```

Comment:
Can you add both settings to the list of default values at the top of the file for documentation purpose?